blueprints/app/files/.eslintignore

```diff
@@ -1,23 +1,13 @@
 # unconventional js
 /blueprints/*/files/
-/vendor/
```

Comment:
Removed from the default blueprint in https://github.com/ember-cli/ember-cli/pull/10050.
Theoretically, should not be needed anymore in new Ember apps.